Tools/clinic/clinic.py

```diff
@@ -6427,8 +6451,11 @@ def format_docstring_parameters(params: list[Parameter]) -> str:
     def format_docstring(self) -> str:
         assert self.function is not None
         f = self.function
-        if f.kind.new_or_init and not f.docstring:
-            # don't render a docstring at all, no signature, nothing.
+        if f.kind in {METHOD_INIT, METHOD_NEW, GETTER, SETTER} and not f.docstring:
+            # If following cases have a empty docstring,
+            # Do not render a docstring at all, no signature, nothing.
+            # 1. __init__ / __new__
+            # 2. @getter / @setter properties
```

Comment:
How about this (so we don't need to repeat what's in the code in the comment)?

```suggestion
        # For the following special cases, it does not make sense to render a docstring.
        if f.kind in {METHOD_INIT, METHOD_NEW, GETTER, SETTER} and not f.docstring:
```

Reply:
Good!